.travis.yml

```
@@ -0,0 +1,60 @@
+dist: xenial
+sudo: required
+
+stages:
+  - Build
+  - Tests
+
+cache:
+  timeout: 1000
+  directories:
+    - $CACHE_DIR
+
+env:
+  global:
+    - CACHE_DIR=$HOME/.cache/docker
+    - CACHE_FILE_IMAGE=$CACHE_DIR/php-fpm.tar.gz
+    - ES_VERSION=6.3.2
+
+before_install:
+  - if [ -f ${CACHE_FILE_IMAGE} ]; then gunzip -c ${CACHE_FILE_IMAGE} | docker load; fi
```

```suggestion
  - if [ -f ${CACHE_FILE_IMAGE} ]; then docker load < ${CACHE_FILE_IMAGE}; fi
```
what about this way ?